src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 14.6px;
+}
+
+.container {
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding for links instead
```suggestion
```